src/main/java/model/process/FromProcess.java

```diff
@@ -11,14 +17,29 @@
  */
 public class FromProcess extends DataProcess {
 
-	private Identifier<DataTable> identifier;
+	private Identifier<DataTable>[] identifiers;
 
-	public FromProcess(Identifier<DataTable> identifier) {
-		this.identifier = identifier;
+	public FromProcess(Identifier<DataTable>... identifiers) {
+		this.identifiers = identifiers;
 	}
 
 	@Override
 	protected Table doProcess() {
-		return getDataModel().getByName(identifier.getName()).copy();
+		Supplier<Stream<DataTable>> tableStream = () ->
+				Arrays.stream(identifiers)
+				.map(x -> getDataModel().getByName(x.getName()).copy());
+		Optional<DataTable> first = tableStream.get().findFirst();
+		DataTable[] tables = tableStream.get().skip(1).toArray(DataTable[]::new);
+
+		if (!first.isPresent()) {
+			throw new IllegalArgumentException(
```

Comment:
Je hebt gelijk. Hij kan alleen voorkomen als identifiers een lege array is, maar dan klopt mijn message niet.

Het is trouwens wel zo dat ik in de setCodes branch de getByName heb aangepast, dus die returnt nu null als hij er geen kon vinden, zoals hoort volgens de JavaDoc.